assets/queries/openAPI/paths_object_empty/test/negative2.yaml

```diff
@@ -0,0 +1,30 @@
+openapi: 3.0.0
+info:
+  title: Simple API overview
+  version: 1.0.0
+paths:
+  "/":
+    get:
+      operationId: listVersionsv2
+      summary: List API versions
+      responses:
+        '200':
+          description: 200 response
+          content:
+            application/json:
+              examples:
+                foo:
+                  value:
+                    versions:
+                    - status: CURRENT
+                      updated: '2011-01-21T11:33:21Z'
+                      id: v2.0
+                      links:
+                      - href: http://127.0.0.1:8774/v2/
+                        rel: self
+security:
```

Comment:
I think security and components can be removed from this example, since it is optional and does not affects query example...